### DISCUSSION TITLE
Instructions to skip domain validation on VM are unclear

### DISCUSSION THREAD

**Discussion by WofWca** [nextcloud/all-in-one#1878] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/39462442/215769063-ce59da8c-3d6d-46d3-9bce-c920c5054ee7.png)

It links [here](https://github.com/nextcloud/all-in-one#how-to-skip-the-domain-validation), but it says

> How to skip the domain validation?
> If you are completely sure that you've configured everything correctly and are not able to pass the domain validation, you may skip the domain validation by adding -e SKIP_DOMAIN_VALIDATION=true to the docker run command of the mastercontainer.

but I'm not the one who ran the docker command, it was run by the VM automatically, and I don't know where to change it.

**Comment by szaimen** [comment]
Hi, are you running into a specific issue when you enter your domain? What does it report?

**Reply by WofWca** [reply]
Hi!
I'm just trying to check out a local installation of Nextcloud, just to see what it's like. I have not tried setting up a domain name.

**Comment by szaimen** [comment, marked as answer]
Edit: the AIO VM is not suitable to run it locally. 

See https://github.com/nextcloud/all-in-one/blob/main/local-instance.md what needs to be done to run it locally.

**Reply by WofWca** [reply]
Thanks, I'll try it!
But it doesn't change the fact that the process is not straight-forward.
